### ISSUE TITLE
Undeniable + player? against

### ISSUE THREAD

**Issue by arserina** [PathOfBuildingCommunity/PathOfBuilding#6710]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the behaviour is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the behaviour in-game?

First of all, I would like to inform you that I used a translator as I am not an English speaker.

I would like to make you aware of the following problem.
[Juggernaut - Undeniable]
The attack speed of the node increases depending on the Accuray Rating.

The problem is this.
Accuracy Mastery - 40% increased Accuracy Rating againt Unique Enemies

This option did not work with [Undeniable] and in-game play
Accuracy Rating did not increase or Attack Speed increased.

In my opinion, I don't think the player's Accuracy Rating increased because it was against Enemies.

[Undeniable] seems to work based on player stats.

The problem is that when you check the option in PoB, the Accuracy Rating and Attack Speed due to Undeniable increase.

There is another similar option.
Mark Mastery - 50% increased Accuracy Rating against Marked Enemy

If this doesn't work, I believe there is also a problem with Oskarm, Stormrider Boots, Obscurantis, Divergent Additional Accuracy Support, etc.

I think there is a difference between the value calculated based on me and the value calculated based on my enemy.

If anyone knows anything about this, please leave a comment.

https://youtu.be/9h1-i29GSmU?t=28153
(korean)

### What is the behaviour in Path of Building?

I wrote the post above.

### Character build code

```shell
https://pobb.in/MIi7otIM8nEB
```

**Comment by Paliak** [comment]
Seems related to #6690

**Comment by Wires77** [comment]
Yeah, it's a duplicate, not sure why the other one is closed, though, so I'll just leave this one open
